--- a/shutdowncheck	2015-01-06 16:08:47.000000000 -0500
+++ b/shutdowncheck	2015-01-06 16:19:50.777612285 -0500
@@ -1,22 +1,24 @@
 #!/bin/bash
 
-PATH=/usr/bin:/home/pi/wiringPi/gpio
+# wpi pin naming convention
+PIN_IN=16
+PIN_OUT=1
 
 echo "ATXRaspi shutdown script starting..."
-echo "Asserting pins (7=in,low; 8=out,high)"
+echo "Asserting pins ($PIN_IN=in,low; $PIN_OUT=out,high)"
 
-gpio -g mode 7 in
-gpio -g write 7 0
-gpio -g mode 8 out
-gpio -g write 8 1
+gpio mode $PIN_IN in
+gpio write $PIN_IN  0
+gpio mode $PIN_OUT out
+gpio write $PIN_OUT 1
 
 while [ 1 ]; do
-  if [ "$(/home/pi/wiringPi/gpio/gpio -g read 7)" = "1" ]; then
-    echo "PIN7 requested a SYSTEM HALT !"
-    sudo poweroff
+  if [ `gpio read $PIN_IN` = 1 ]; then
+    echo "$PIN_IN requested a SYSTEM HALT !"
+    poweroff
     break
   fi
-  /bin/sleep 0.5
+  /bin/sleep 1
 done
 
-exit 0
\ No newline at end of file
+exit 0

