--- a/gpio/Makefile	2014-12-28 12:30:25.034942598 -0500
+++ b/gpio/Makefile	2014-12-28 15:43:07.365498924 -0500
@@ -27,12 +27,12 @@
 PREFIX=/local
 
 #DEBUG	= -g -O0
+#CC = $(CC)
 DEBUG	= -O2
-CC	= gcc
-INCLUDE	= -I$(DESTDIR)$(PREFIX)/include
+INCLUDE	= -I$(DESTDIR)$(PREFIX)/include -I../wiringPi -I../devLib
 CFLAGS	= $(DEBUG) -Wall $(INCLUDE) -Winline -pipe
 
-LDFLAGS	= -L$(DESTDIR)$(PREFIX)/lib
+LDFLAGS	= -L$(DESTDIR)$(PREFIX)/lib -L../wiringPi -L../devLib
 LIBS    = -lwiringPi -lwiringPiDev -lpthread -lm
 
 # May not need to  alter anything below this line
@@ -46,6 +46,7 @@
 
 gpio:	$(OBJ)
 	@echo [Link]
+	@echo $(CC) -o $@ $(OBJ) $(LDFLAGS) $(LIBS)
 	@$(CC) -o $@ $(OBJ) $(LDFLAGS) $(LIBS)
 	
 .c.o:
